Src/AutoFakeItEasyUnitTest/AutoFakeItEasyCustomizationTest.cs

```diff
@@ -63,6 +63,16 @@ public void InitializeWithGenerateDelegatesOptionThrowsIfNotSupported()
         }
 #endif
 
+        [Fact]
+        public void ConfigureMembersIsDisabledByDefault()
```

Comment:
Could you please add a set of kinda structural tests like this: https://github.com/AutoFixture/AutoFixture/blob/f2378ec82e92f124ddac8468a545fa04bf9eddfe/Src/AutoNSubstituteUnitTest/AutoNSubstituteCustomizationTest.cs#L178

To unit-test the customization. The test below are "facade" or "integration" ones.